Makefile

```diff
@@ -1,6 +1,9 @@
 postgres:
-	docker run --name postgres15 -p 5432:5432 -e POSTGRES_USER=root -e POSTGRES_PASSWORD=password -d postgres:latest
-
+	# docker pull jesselee2020/bank_backend_postgres:latest
+	# docker tag jesselee2020/bank_backend_postgres:latest bank_backend_postgres
```

Comment:
Comments need to be removed before pushing into production in future.

Reply:
Updated